modify-and-filter-traffic/replaying-traffic.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Replaying Rules
+description: "Replay HTTPS sessions simultaneously or consiquencily while using Fiddler Everywhere advanced replay options"
+slug: replaying-traffic
+publish: true
+position: 25
+---
+
+# Replaying Traffic
+
+When developing and testing web applications, it's common to wonder how your server and site will handle many continuous requests. You may also want to test if a specific functionality of your web application will respond correctly to multiple repeating requests. With Fiddler Everywhere, you can replay a session or multiple sessions and set specific repeating conditions such as the number of repeats, repeat behavior, and request behavior for each separate repeat.
+
+## One-Time Replay
+
+To trigger a one-time replay for selected sessions, follow these steps:
+
+1. Select one or more sessions.
+2. Invoke the context menu or press `R`.
+
+As a result, the selected sessions will be immediately replayed and displayed as new entries in the Fiddler's traffic grid.
+
+![One-time replay for sessions](../images/rules/replay-simple.png)
+
+## Advanced Replay
+
+For more advanced replay options, follow these steps:
+
+1. Select one or more sessions.
+2. Invoke the context menu or press `Alt + R`/`Control + R`.
```

Comment:
I am not sure if we have to mention it explicitly for all OSes, but for Mac it is Option + R